### DISCUSSION TITLE
[Feature Request] Actual Budget widget support

### DISCUSSION THREAD

**Discussion by gael-vanderlee** [gethomepage/homepage#4302] [Feature Requests]
### Description

[Actual budget](https://actualbudget.org/) is an app to manage your finances.

API references are [here](https://actualbudget.org/docs/api/reference/) and [here](https://actualbudget.org/docs/api/actual-ql/)

Would be nice to know the state of the current month's budget or latest transactions.

This is reopening a [previous discussion](https://github.com/gethomepage/homepage/discussions/1302?sort=new) which was closed due to inactivity. Actual budget has gained a lot of traction since

**Comment by theMajc** [comment]
Second this. And to anyone interested, don't forget to click the :up vote for anyone to be able to work on this, as the [service widget guidelines](https://gethomepage.dev/more/development/#service-widget-guidelines) requires at least 20 votes for the feature request to be considered.

**Comment by matheusvellone** [comment]
Upvoted!
State of the current month also looks like a cool info to be displayed
Also, having the uncategorized transactions count on homepage would be very helpful for me, since it indicates that are pending tasks waiting for me on Actual.

I was taking a look at the ["Actual API"](https://actualbudget.org/docs/api/), and its a non-standard way to implement it, since there are no HTTP endpoints documented.
Couldn't find a way to properly query data without installing the SDK yet 😕

**Reply by servaasvdc** [reply]
> Upvoted! State of the current month also looks like a cool info to be displayed Also, having the uncategorized transactions count on homepage would be very helpful for me, since it indicates that are pending tasks waiting for me on Actual.
> 
> I was taking a look at the ["Actual API"](https://actualbudget.org/docs/api/), and its a non-standard way to implement it, since there are no HTTP endpoints documented. Couldn't find a way to properly query data without installing the SDK yet 😕

https://github.com/jhonderson/actual-http-api could be a solution, but definitely a lot of work for just a widget

**Comment by crispybegs** [comment]
would love this also

**Comment by servaasvdc** [comment]
Doesn't look like they're planning a REST API: https://github.com/actualbudget/actual/issues/1384
